src/variables/utils/getWindowVars.test.ts

```diff
@@ -21,6 +21,26 @@ describe('getWindowPeriodVariable', () => {
     expect(actual).toEqual(null)
   })
 
+  test('should return null when timeRange is identical', () => {
```

Comment:
one thing you might do for SUPER EXTRA PROTECTION AND DOCUMENTATION™ is to add a comment linking to the issue this test is guarding against. so if i'm bumbling around in tests and i see this one and i see a link to a bug, i can get more context if i'm so inclined.

Reply:
Your wish is my command, and so it is done